src/screens/ProfileScreen/elements/RenderItem.js

```diff
@@ -247,6 +252,7 @@ const Item = ({
             });
           }}
           statusVote={voteStatus}
+          isShowDM
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

A new prop `isShowDM` is added without any value. This might cause issues if the child component expects a boolean value for this prop. Consider providing a default value or ensuring the parent component always passes a valid value.
```diff
+ isShowDM
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->